src/provider/store/Actions.ts

```diff
@@ -3,25 +3,20 @@ import {StoredApplication} from '../model/Environment';
 import {ToggleFilter} from '../utils/ToggleFilter';
 
 import {RootState} from './State';
-import {StoreAPI, AsyncAction, Action} from './Store';
+import {Action, StoreAPI} from './Store';
+
+export interface Actionable {
+    storeApi: StoreAPI<RootState>;
+}
 
 export const enum ToggleCenterVisibilitySource {
     API,
     TRAY,
     BUTTON
 }
 
-export type RootAction =
-    CreateNotification |
-    RemoveNotifications |
-    ClickNotification |
-    ClickButton |
-    ToggleCenterVisibility |
-    BlurCenter |
-    ToggleLockCenter |
-    RegisterApplication;
-
-export class CreateNotification extends AsyncAction<RootState> {
+export class CreateNotification extends Action<RootState> {
+    public readonly type = '@@NOTIFICATION/CREATE_NOTIFICATION';
```

Comment:
Since we're not using real redux, is there any value to following redux convention here?

Reply:
At some point I do want to try and get devtools working with our implementation, and the prefix is nice so you know what property on the state an action should/can modify.